.github/workflows/lint.yml

```diff
@@ -116,3 +116,9 @@ jobs:
     - name: lint remain
       run: |
         make lint-remain
+
+    - name: install markdownlint cli
+      run: npm install -g markdownlint-cli
+
+    - name: lint markdown
+      run: markdownlint '**/*.md' --disable MD012 MD013 MD029 MD033 MD034 MD036 MD041
```

Comment:
Would you move this cmd into the Makefile so that the developer can run `make lint-markdown` to lint locally?